src/Microsoft.IdentityModel.Logging/LogHelper.cs

```diff
@@ -371,5 +417,47 @@ public static object MarkAsNonPII(object arg)
         {
             return new NonPII(arg);
         }
+
+        /// <summary>
+        /// Creates a <see cref="LogEntry"/> by using the provided event level, exception argument, string argument and arguments list.
+        /// </summary>
+        /// <param name="level"><see cref="EventLevel"/></param>
+        /// <param name="innerException"><see cref="Exception"/></param>
+        /// <param name="message">The log message.</param>
+        /// <param name="args">An object array that contains zero or more objects to format.</param>
+        private static LogEntry WriteEntry(EventLevel level, Exception innerException, string message, params object[] args)
+        {
+            if (innerException != null)
+            {
+                // if PII is turned off and 'innerException' is a System exception only display the exception type
+                if (!IdentityModelEventSource.ShowPII && !LogHelper.IsCustomException(innerException))
```

Comment:
This is tightly coupled to `IdentityModelEventSource`

Reply:
entry.Message = args != null ? FormatInvariant(message, args) : message;

True and but being a static its available to the LogHelper class and FormatInvariant uses it to evaluate if arguments need to be scrubbed.

Reply:
@bgavrilMS are you suggesting the IIdentityLogger should have a ShowPII property?

Reply:
I was suggesting this originally, but then @sruke did point out that separating PII concept from Logging concept is better long term and easier to integrate. So I think it's fine that IIdentitityLogger remains de-coupled from PII and each of our libs does PII in their own way (SAL via the static, MSAL via a user config). 

Ultimately we want to get to a point where the app developer can configure: 

- an IIdenittyLogger 
- a bool PII (simple, i.e. I don't want PII in my logs) or a log scrubber (fine grained control of PII)